### ISSUE TITLE
Doesn't appear to account for custom table names

### ISSUE THREAD

**Issue by secondstreetmedia** [trptcolin/consistency_fail#32]
Some of my models are broken up into namespaces.  In order for me to more easily differentiate my database tables, I have prefixed table names with their namespace. For example: 
```ruby
class Acme::Project < ActiveRecord::Base
  self.table_name = 'acme_projects'
```
But for `has_one` relationships, consistency_fail will look for a unique index attached to `project_id` rather than `acme_project_id`.  Is there a way, when `self.table_name` is used, to take it into account?

Thanks,
Tim

**Comment by trptcolin** [comment]
This issue is fixed in the 0.3.4 release, thanks to @fw42's fix from #21.

**Comment by trptcolin** [comment]
(Thanks for reporting it!)
